src/app/bucket/bucket.component.ts

```diff
@@ -43,6 +43,7 @@ export class BucketComponent implements OnInit {
   @Output() onMoveObjectiveBucket = new EventEmitter<[ImmutableObjective, ImmutableBucket, ImmutableObjective, ImmutableBucket]>();
   @Output() onChanged = new EventEmitter<[ImmutableBucket, ImmutableBucket]>();
   @Output() onDelete = new EventEmitter<ImmutableBucket>();
+  @Input() cumilativeSum?:number=0;
```

Comment:
Please add this at the end of the list of `@Input()`s, so that all the inputs are together and all the outputs are together.

Also: do we need an explicit default here? I'd expect this to be defined on every objective because of the way they are constructed.